customer-account-extension/shopify.extension.toml.liquid

```diff
@@ -6,5 +6,29 @@ name = "{{ name }}"
 handle = "{{ handle }}"
 
```

Comment:
Don't we need to update the `api_version` at the top of this file to be "unstable"?

Reply:
Also, checkout includes the `api_access` capability [by default](https://github.com/Shopify/extensions-templates/blob/main/checkout-extension/shopify.extension.toml.liquid#L24), we should probably do the same. And [add commented out config](https://github.com/Shopify/extensions-templates/blob/main/checkout-extension/shopify.extension.toml.liquid#L29) for `network_access` with a link to our docs